wi-fi/lwip/main.c

```diff
@@ -67,21 +79,54 @@ static ip_static_addr_t ap_addr = {
 };
 
 
-int wifi_ap_set_ssid(const char *ssid, size_t len)
+static int wifi_ap_set_idle_timeout(const char *data, size_t len)
+{
+	char buf[16];
+	long int timeout;
+
+	if (len > (sizeof(buf) - 1)) {
+		wm_cy_log_msg(CYLF_MIDDLEWARE, CY_LOG_ERR, "can't set Wi-Fi idle timeout (max length is %u)\n", sizeof(buf) - 1);
+		return -1;
+	}
+
+	memcpy(buf, data, len);
+	buf[len] = '\0';
+
+	timeout = strtol(buf, NULL, 0);
+	if (timeout < 0) {
+		wm_cy_log_msg(CYLF_MIDDLEWARE, CY_LOG_ERR, "can't set Wi-Fi idle timeout (min value is 0)\n");
+		return -1;
+	}
+	else if (timeout > UINT32_MAX) {
```

Comment:
Hmmm, what if long == int? 
```suggestion
	else if (timeout > INT32_MAX) {
```

Reply:
If long == int condition is always false which is no problem. Are you worried about compiler warning?

Reply:
Right, no problem here, sorry